### PR TITLE
Styling adjustment on Test History for Firefox

### PR DESCRIPTION
**Pull request by v-hclaveria**
Adding/adjusting CSS classes to improve UI to look better on Firefox browser

### CODE REVIEW

File: www/resources/views/pages/testhistoryadmin.blade.php [hunk 1]
```diff
@@ -72,7 +72,7 @@
                         'labelTxt' => $labelTxt,
                     ])
                 <tr>
-                    <th>
+                    <th class="history_checkbox">
```
Comment (claud-io) [inline www/resources/views/pages/testhistoryadmin.blade.php:75]: why history_checkbox on a th?

Comment (v-hclaveria) [reply www/resources/views/pages/testhistoryadmin.blade.php:75]: It's meant to align the content. Maybe a better name like 'select_test' or something?

File: www/resources/views/pages/testhistory.blade.php [hunk 2]
```diff
@@ -35,34 +35,31 @@
...
-                                <td class="label">{{ $record->getLabel() }}</td>
-                            </tr>
-                        @endforeach
-                    </tbody>
-                @endif
-            </table>
-        </div>
-
-
+        <table id="history" class="history pretty" border="0" cellpadding="5px" cellspacing="0">
```
[the remark this answers is not in the document]
Comment (v-hclaveria) [reply www/resources/views/pages/testhistory.blade.php]: Added

File: www/resources/views/pages/testhistoryadmin.blade.php (hunk 1 above)
Comment (claud-io) [reply www/resources/views/pages/testhistoryadmin.blade.php:75]: yes please let's find a better name

File: www/resources/views/pages/testhistoryadmin.blade.php [hunk 3]
```diff
@@ -89,7 +89,7 @@
                             </b>
                         @endif
                     </td>
-                    <td class="location">{!! $location !!}
+                    <td class="location scrollable-td"><div class="scrollable-td-content">{!! $location !!}</div>
```
Comment (claud-io) [inline www/resources/views/pages/testhistoryadmin.blade.php:92]: @v-hclaveria is this expected that the below span is outside the div?

Comment (claud-io) [reply www/resources/views/pages/testhistoryadmin.blade.php:92]: please also ensure that those changes are not impacting table on other browsers

Comment (v-hclaveria) [reply www/resources/views/pages/testhistoryadmin.blade.php]: image

File: www/resources/views/pages/testhistory.blade.php [hunk 4]
```diff
@@ -50,19 +50,18 @@
...
                                 <td class="label">{{ $record->getLabel() }}</td>
                             </tr>
                         @endforeach
                     </tbody>
                 @endif
             </table>
         </div>
-
-
+
```
Comment (claud-io) [inline www/resources/views/pages/testhistory.blade.php:64]:
```suggestion
```

File: www/resources/views/pages/testhistory.blade.php [hunk 5]
```diff
@@ -50,19 +50,17 @@
...
                                 <td class="date">{{ date_format(date_create($record->getStartTime()), 'M d, Y g:i:s A e') }}</td>
-                                <td class="location">{{ $record->getLocation() }}</td>
+                                <td class="location scrollable-td"><div class="scrollable-td-content">{{ $record->getLocation() }}</div></td>
                                 <td class="label">{{ $record->getLabel() }}</td>
                             </tr>
                         @endforeach
                     </tbody>
                 @endif
             </table>
         </div>
```
Comment (claud-io) [inline www/resources/views/pages/testhistory.blade.php:63]:
```suggestion
        </div>
        
        
```

File: www/resources/views/pages/testhistory.blade.php [hunk 6]
```diff
@@ -50,19 +50,19 @@
...
-
-
+
```
Comment (claud-io) [inline www/resources/views/pages/testhistory.blade.php:64]:
```suggestion

```

**Review by claud-io — APPROVED** [review]

**Pull request merged**